packages/block-library/src/gallery/gallery-image.js

```diff
@@ -127,7 +128,7 @@ class GalleryImage extends Component {
 		// Disable reason: Each block can be selected by clicking on it and we should keep the same saved markup
 		/* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/onclick-has-role, jsx-a11y/click-events-have-key-events */
 		return (
-			<figure className={ className } tabIndex="-1" onKeyDown={ this.onKeyDown } ref={ this.bindContainer }>
+			<figure className={ className }>
```

Comment:
Is code comment explaining disable reason still valid?

Reply:
Was not! Thank you. Fixed at be94811